listenbrainz/webserver/views/player.py

```diff
@@ -74,6 +74,9 @@ def load_instant():
     the recording_mbid's metadata and creates a JSPF file from this data and sends it to the front end
     so a playlist can be instantly played.
 
+    NOTE: We recommend that you do not send more than 50 recording_mbids in one request -- our
```

Comment:
If you write using `.. note::`, 
https://github.com/metabrainz/listenbrainz-server/blob/9e315466678033071c9ad5cf98d7abda8a6ccbb5/listenbrainz/webserver/views/feedback_api.py#L206-L208

reathedocs will render it nicely in a [blue box](https://listenbrainz.readthedocs.io/en/latest/users/api/recordings/#get--1-feedback-user-(user_name)-get-feedback-for-recordings) that stands out.